Upgrading from Symfony 2.8 to 3.0

# Question
I could run the prefixed version (Rector v0.7.27) as 'vendor/rector/rector-prefixed/rector process src --set symfony28 --dry-run' successfully on my v2.8 application. It went well, however if I want to move forward and upgrade to Symfony 3.0 with '--set symfony30', I get the following error message:

`
 [ERROR] Could not process "src/project/AppBundle/Controller/DefaultController.php" file, due to:                      
         "File path                                                                                                     
         "phar:///<projectDir>/vendor/rector/rector-prefixed/rector/src/Rector/AbstractRector/NameResolverTrait.php" was not found while creating                                                      
         "_HumbugBox3c08e8add8e1\Symplify\SmartFileSystem\SmartFileInfo" object.".    
`

Did I miss anything?

--- a/packages/better-php-doc-parser/src/PhpDocNode/AbstractTagValueNode.php
+++ b/packages/better-php-doc-parser/src/PhpDocNode/AbstractTagValueNode.php
@@ -12,6 +12,7 @@ use Rector\BetterPhpDocParser\Contract\Doctrine\DoctrineTagNodeInterface;
 use Rector\BetterPhpDocParser\Contract\PhpDocNode\AttributeAwareNodeInterface;
 use Rector\BetterPhpDocParser\Contract\PhpDocNode\SilentKeyNodeInterface;
 use Rector\BetterPhpDocParser\Contract\PhpDocNode\TagAwareNodeInterface;
+use Rector\BetterPhpDocParser\PhpDocNode\Sensio\SensioRouteTagValueNode;
 use Rector\BetterPhpDocParser\PhpDocNode\Symfony\SymfonyRouteTagValueNode;
 use Rector\BetterPhpDocParser\Utils\ArrayItemStaticHelper;
 use Symfony\Component\Routing\Annotation\Route;
@@ -151,7 +152,7 @@ abstract class AbstractTagValueNode implements AttributeAwareNodeInterface, PhpD
         $json = Strings::replace($json, '#\\\\\\\\#', '\\');
 
         // replace ":" with "=" for @Route
-        if ($this instanceof SymfonyRouteTagValueNode || $this instanceof DoctrineTagNodeInterface) {
+        if ($this instanceof SymfonyRouteTagValueNode || $this instanceof DoctrineTagNodeInterface || $this instanceof SensioRouteTagValueNode) {
             // @see https://regex101.com/r/XfKi4A/1/
             $json = Strings::replace($json, '#(\"|\w)\:(\"|\w)#', '$1=$2');
         }

--- a/packages/node-name-resolver/src/NodeNameResolver.php
+++ b/packages/node-name-resolver/src/NodeNameResolver.php
@@ -218,8 +218,13 @@ final class NodeNameResolver
         $rectorBacktrace = $this->matchRectorBacktraceCall($backtrace);
 
         if ($rectorBacktrace) {
-            $fileInfo = new SmartFileInfo($rectorBacktrace['file']);
-            $fileAndLine = $fileInfo->getRelativeFilePathFromCwd() . ':' . $rectorBacktrace['line'];
+            // issues to find the file in prefixed
+            if (file_exists($rectorBacktrace['file'])) {
+                $fileInfo = new SmartFileInfo($rectorBacktrace['file']);
+                $fileAndLine = $fileInfo->getRelativeFilePathFromCwd() . ':' . $rectorBacktrace['line'];
+            } else {
+                $fileAndLine = $rectorBacktrace['file'] . ':' . $rectorBacktrace['line'];
+            }
 
             $message .= PHP_EOL . PHP_EOL;
             $message .= sprintf('Look at %s', $fileAndLine);

--- a/packages/node-name-resolver/src/NodeNameResolver.php
+++ b/packages/node-name-resolver/src/NodeNameResolver.php
@@ -24,6 +24,11 @@ use Symplify\SmartFileSystem\SmartFileInfo;
 final class NodeNameResolver
 {
     /**
+     * @var string
+     */
+    public const FILE = 'file';
+
+    /**
      * @var NodeNameResolverInterface[]
      */
     private $nodeNameResolvers = [];
@@ -219,11 +224,11 @@ final class NodeNameResolver
 
         if ($rectorBacktrace) {
             // issues to find the file in prefixed
-            if (file_exists($rectorBacktrace['file'])) {
-                $fileInfo = new SmartFileInfo($rectorBacktrace['file']);
+            if (file_exists($rectorBacktrace[self::FILE])) {
+                $fileInfo = new SmartFileInfo($rectorBacktrace[self::FILE]);
                 $fileAndLine = $fileInfo->getRelativeFilePathFromCwd() . ':' . $rectorBacktrace['line'];
             } else {
-                $fileAndLine = $rectorBacktrace['file'] . ':' . $rectorBacktrace['line'];
+                $fileAndLine = $rectorBacktrace[self::FILE] . ':' . $rectorBacktrace['line'];
             }
 
             $message .= PHP_EOL . PHP_EOL;

--- a/src/Rector/AbstractRector/NameResolverTrait.php
+++ b/src/Rector/AbstractRector/NameResolverTrait.php
@@ -91,6 +91,14 @@ trait NameResolverTrait
             return false;
         }
 
+        if ($node->var instanceof StaticCall) {
+            return false;
+        }
+
+        if ($node->var instanceof MethodCall) {
+            return false;
+        }
+
         if (! $this->isName($node->var, 'this')) {
             return false;
         }

